make/deps.mk

```diff
@@ -1,4 +1,5 @@
-DEPS ?= amqp_client \
```

Comment:
I see a commit message that says you revert the changes to deps.mk but it doesn't say why these changes are made?  PBX-14 was already merged and completed.

Reply:
Understood, but do we need all of these dependencies or is there one specifically that is required?  `apns` and `fcm` are just for doing push notifications with Apple and Google respectively and I don't think will have any impact on this tool.